kernel/nativelib.ml

```diff
@@ -25,7 +25,7 @@ let open_header = ["Nativevalues";
 let open_header = List.map mk_open open_header
 
 (* Directory where compiled files are stored *)
-let output_dir = ".coq-native"
+let output_dir = ref (Some ".coq-native")
```

Comment:
This should be `None` since the the default is set in the configure.

Reply:
Sorry @gares where is this set? I see it is hardcoded in master so far.

Reply:
In this patch, not in master

Reply:
Ah, you mean in the toplevel code; I guess it doesn't matter a lot, people linking this library maybe would like to get this default ?